pkg/sql/parser/sql.y

```diff
@@ -3722,7 +3722,14 @@ typename:
   }
   // SQL standard syntax, currently only one-dimensional
 | simple_typename ARRAY '[' ICONST ']' { return unimplementedWithIssue(sqllex, 2115) }
```

Comment:
@knz is that a question to me about my previous comment? Array bounds are left unimplemented in the `opt_array_bounds` rule as well.